misk-redis/src/main/kotlin/misk/redis/JedisPipeline.kt

```diff
@@ -0,0 +1,78 @@
+package misk.redis
+
+import okio.Closeable
+import redis.clients.jedis.ClusterPipeline
+import redis.clients.jedis.Pipeline
+import redis.clients.jedis.Response
+import redis.clients.jedis.args.FlushMode
+import redis.clients.jedis.args.FunctionRestorePolicy
+import redis.clients.jedis.commands.PipelineBinaryCommands
+import redis.clients.jedis.commands.PipelineCommands
+
+sealed class JedisPipeline : Closeable, PipelineCommands, PipelineBinaryCommands {
+  class ClusterJedisPipeline(private val pipeline: ClusterPipeline) : JedisPipeline(),
```

Comment:
Jedis / UnifiedJedis is a little annoying as there is no unified Pipeline interface.

We have to support JedisClustered and JedisPooled. So... I unified it.